inventory/group_vars/all.yml

```diff
@@ -1,10 +1,29 @@
 ### general settings
 product_version: "6.1.0"
-
+ansible_become: yes
```

Comment:
why do we have this global setting here?

Reply:
I thought we need it to pass privileges to the remote hosts. But I just tested this variable, and it gives root privileges to all commands, not only to these that have become: yes. I purged this variable and tested successfully (not on a distributed setup)